lib/model.js

```diff
@@ -3365,24 +3390,43 @@ class Model {
       if (!options.fields.length) return this;
       if (!this.changed() && !this.isNewRecord) return this;
 
+      const versionFieldName = _.get(this.constructor.rawAttributes[versionAttr], 'field') || versionAttr;
```

Comment:
The `get` protects against accessing a property on an undefined field.  To remove the get, it would have to be:
`const versionFieldName = this.constructor.rawAttributes[versionAttr] ?  this.constructor.rawAttributes[versionAttr].field : versionAttr;`